### PR TITLE
Add a --generate-code flag to import

### PR DESCRIPTION
**Pull request by Frassle**
# Description

Adds a flag `--generate-code` to pulumi import. By default this is true, but users can pass `--generate-code=false` to skip the code generation step of import.

Fixes https://github.com/pulumi/pulumi/issues/9134

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works

- [x] I have updated the [CHANGELOG-PENDING](https://github.com/pulumi/pulumi/blob/master/CHANGELOG_PENDING.md) file with my change

- [ ] Yes, there are changes in this PR that warrants bumping the Pulumi Service API version

### CODE REVIEW

File: pkg/cmd/pulumi/import.go [hunk 1]
```diff
@@ -496,37 +501,39 @@ func newImportCmd() *cobra.Command {
 				Scopes:             cancellationScopes,
 			}, imports)
 
-			deployment, err := getCurrentDeploymentForStack(s)
-			if err != nil {
-				return result.FromError(err)
-			}
+			if generateCode {
```
Comment (pgavlin) [inline pkg/cmd/pulumi/import.go:504]: Minor nit: maybe pull the body of this `if` out into its own function? Normally I'd suggest using an early return here to be more idiomatic and avoid the rightward movement, but that isn't convenient with the `res != nil` bits later.

File: pkg/cmd/pulumi/import.go [hunk 2]
```diff
@@ -378,6 +379,10 @@ func newImportCmd() *cobra.Command {
 				importFile = f
 			}
 
+			if !generateCode && outputFilePath != "" {
+				return result.Errorf("an output file may not be specified in conjunction with --generate-code=false")
+			}
```
Comment (pgavlin) [inline pkg/cmd/pulumi/import.go:384]: Nit: seems like we could just make this a warning rather than an error.

**Review by pgavlin — APPROVED** [review]
Two nits, LGTM otherwise. Thanks for turning this around so fast!

File: pkg/cmd/pulumi/import.go [hunk 3]
```diff
@@ -378,6 +379,10 @@ func newImportCmd() *cobra.Command {
 				importFile = f
 			}
 
+			if !generateCode && outputFilePath != "" {
+				fmt.Print("Output file will not be used as --generate-code is false.\n")
```
Comment (iwahbe) [inline pkg/cmd/pulumi/import.go:383]: I would print this message to stderr.

File: pkg/cmd/pulumi/import.go (hunk 1 above)
Comment (Frassle) [reply pkg/cmd/pulumi/import.go:504]: I'd generally lean towards keeping the code inline here, we're just hitting 120 cols which doesn't seem unreasonable.

**Review by iwahbe — APPROVED** [review]

**Pull request merged**